scripts/storage/secondary/setup-sysvm-tmplt

```diff
@@ -146,9 +146,9 @@ fi
 
 
 tmpltfile=$destdir/$localfile
-tmpltsize=$(ls -l $tmpltfile | awk -F" " '{print $5}')
+tmpltsize=$(sudo ls -l $tmpltfile | awk -F" " '{print $5}')
```

Comment:
as teh `touch $tmplfile` at line 111 is without `sudo`, does this need it?

Reply:
@DaanHoogland needed it here, as tmpltfile is in destdir created with sudo in line 93.

Reply:
ok, than I don't understand why `sudo` is not needed on line 111, but if it tests alright then alright.

Reply:
I just noticed it @DaanHoogland L111 the variable is `tmplfile` and here it's `tmpltfile` - pointing to 2 different file paths.

Reply:
tnx @Pearl1594